src/main/java/appeng/util/item/AEItemStack.java

```diff
@@ -186,20 +184,14 @@ public static IAEItemStack loadItemStackFromPacket( final ByteBuf data ) throws
 
 		final ItemStack itemstack = new ItemStack( d );
 
-		if( !showCraftingLabel )
-		{
-			showCraftingLabel = stackSize == 0 && isCraftable;
-		}
-
 		if( itemstack.isEmpty() )
 		{
 			return null;
 		}
 
 		final AEItemStack item = AEItemStack.create( itemstack );
 		// item.priority = (int) priority;
-		item.setStackSize( showCraftingLabel ? 1 : stackSize );
-		item.setShowCraftingLabel( showCraftingLabel );
+		item.setStackSize( stackSize );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') A "NullPointerException" could be thrown; "item" is nullable here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS2259)